src/components/vuestic-components/va-file-upload/VaFileUploadGalleryItem.vue

```diff
@@ -87,13 +85,20 @@ export default {
       this.removed = false
     },
     convertToImg () {
-      const reader = new FileReader()
-      const imageFileTypes = ['/png', '/jpg', '/jpeg', '/gif']
-      reader.readAsDataURL(this.file.image)
-      reader.onload = (e) => {
-        for (let i = 0; i < imageFileTypes.length; i++) {
-          if (e.target.result.indexOf(imageFileTypes[i]) >= 0) {
-            this.previewImage = e.target.result
+      if (!this.file.name) {
+        return
+      }
+      if (this.file.image && this.file.image.url) {
+        this.previewImage = this.file.image.url
+      } else {
+        const reader = new FileReader()
+        const imageFileTypes = ['/png', '/jpg', '/jpeg', '/gif']
```

Comment:
What about different file formats?
For example `tiff` comes to mind. Probably makes sense to do a small investigation on this.

Reply:
I agree. It's necessary to add at least default image if the imageFileTypes list doesn't include the current type. I've created another issue for it #20